pkg/cmd/release-payload-controller/payload_verification_controller.go

```diff
@@ -159,6 +162,10 @@ func (c *PayloadVerificationController) sync(ctx context.Context, key string) er
 		releasePayload.Status.InformingJobResults = append(releasePayload.Status.InformingJobResults, generateJobStatus(verify))
 	}
 
+	releasePayload.Status.ReleaseCreationJobResult = v1alpha1.ReleaseCreationJobResult{
```

Comment:
Can this also set
```go
		Coordinates: v1alpha1.ReleaseCreationJobCoordinates{
			Name:      originalReleasePayload.Name,
			Namespace: c.jobsNamespace,
		},
		Status: v1alpha1.ReleaseCreationJobUnknown,
```

Reply:
Moved the setting of these fields into their own controllers:
`ReleaseCreationJobController` and `ReleaseCreationStatusController`